### DISCUSSION TITLE
iomega ZIP drive support included?

### DISCUSSION THREAD

**Discussion by Firminator** [PartialVolume/shredos.x86_64#100] [Q&A]
Wondering if I could wipe 100MB ZIP disks with ShredOS. Pushing the boundaries here :)
Has anyone tested or was successful yet? For floppy disks I think it's now established that the Gutmann method actually makes sense to use.

**Comment by Firminator** [comment]
On another thought: are Floppy Disks / Floppy Disk Controllers detected with ShredOS and can you wipe floppies?

**Reply by PartialVolume** [reply]
> On another thought: are Floppy Disks / Floppy Disk Controllers detected with ShredOS and can you wipe floppies?

I have a P4 with a 3.5" floppy drive, I've never actually tried wiping a floppy !. Not sure I've even got any 3.5" floppies here at the moment.

I think pushing the limits would be a 8" floppy drive but you never know.

**Comment by PartialVolume** [comment]
> Wondering if I could wipe 100MB ZIP disks with ShredOS. Pushing the boundaries here :) Has anyone tested or was successful yet? For floppy disks I think it's now established that the Gutmann method actually makes sense to use.

I do actually have some 100MB zip drives and brand new (about 20 years old) zip media still in it's original shrink wrap. I've never tried wiping a zip drive though.

**Comment by Firminator** [comment]
I need to wipe 3 ZIP disks. Trying to figure out how without going through the hassle of installing an OS.
The ZIP drive is IDE with ATAPI mode 3 accd. to the POST screen. Rabbithole, but nonethless interesting.

**Comment by Firminator** [comment]
The ZIP drive is IDE, so that in itself is already a challenge to get it connected. My wiping box doesn't have IDE, so I needed to test adapters. Luckily I got the ZIP drive now recognized: It shows in the BIOS as detected SATA drive by using the SYBA SD-ADA50016 IDE-to-SATA Adapter (https://www.sybausa.com/index.php?route=product/product&product_id=191). However ShredOS boots and tells me there is no device attached.
`lsblk` doesn't show anything either.
`dmesg` shows the drive is being detected by the kernel so that's working.
